### DISCUSSION TITLE
Appflowy API ?

### DISCUSSION THREAD

**Discussion by LoganTann** [AppFlowy-IO/AppFlowy#258] [Ideas]
Hello, 
This is a very nice project and I plan to try it soon !

Does Appflowy provides an API, and if not, is it a planned feature ?

I've seen Notion is sometimes used as a rich text editor with nested pages, that can be used to store a blog or a bot's strings for example.

**Comment by LoganTann** [comment]
Oh nevermind, I thought appflowy was a web app that could be self-hosted, but seems like it's currently a native note-taking app where collaboration is still not implemented

**Comment by Neuxbane** [comment]
It would be better if the AppFlowy has a function to get data from certain API or push or get data to certain API like Notion...
